fontir/src/serde.rs

```diff
@@ -1,99 +1,46 @@
-use std::{collections::HashMap, path::PathBuf};
+use std::{collections::HashSet, path::PathBuf};
 
-use filetime::FileTime;
 use serde::{Deserialize, Serialize};
 
-use crate::stateset::{FileState, MemoryState, State, StateIdentifier, StateSet};
+use crate::stateset::{PartialSource, PartialSourceIdentifier};
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
-pub(crate) struct StateSetSerdeRepr {
-    files: Vec<FileStateSerdeRepr>,
-    slices: Vec<SliceStateSerdeRepr>,
+pub(crate) struct PartialSourceSerdeRepr {
```

Comment:
I forget the details but this whole module might no longer be necessary?

Reply:
Getting there :)

Reply:
...gone